spec/services/bulk_upload/sales/year2024/row_parser_spec.rb

```diff
@@ -1017,14 +1017,6 @@ def purch_id_field
         end
       end
 
-      context "when it's not shared ownership" do
-        let(:attributes) { setup_section_params.merge(field_8: "2", field_103: "3", field_86: "1", field_87: "50", field_88: "99", field_109: nil) }
-
-        it "does not add errors" do
-          expect(parser.errors[:field_103]).not_to include("Enter a valid value for Was a mortgage used for the purchase of this property? - Shared ownership")
-        end
-      end
-
       context "when value is 3 and stairowned is 100" do
```

Comment:
I think we need a "when value is 3 and stairowned is  not 100" test here as well!

Reply:
I think there is one on line 1004 (?)